### DISCUSSION TITLE
Creating a playground

### DISCUSSION THREAD

**Discussion by jucasoliveira** [jucasoliveira/terminalGPT#56] [Announcements]
Hi Everyone, I'd like to thank you all for the support on the terminalGPT. I saw there is some new errors and I'll be addressing them soon.

I want more ppl to be able to use terminalGPT before they install it, so I'll be creating a playground where you will be able to use the library on the browser!

Thanks again for the support!!

**Comment by gianpaj** [comment]
great idea!
